history/history-impl/src/main/java/com/duckduckgo/history/impl/RealNavigationHistory.kt

```diff
@@ -42,4 +42,8 @@ class RealNavigationHistory @Inject constructor(
     override fun getHistorySingle(): Single<List<HistoryEntry>> {
         return historyRepository.getHistoryObservable()
     }
+
+    override suspend fun clearHistory() {
```

Comment:
NIT: if we make it suspend, it's best if we enforce dispatcher here. Just adding `withContext(dispatcher.io())`

Reply:
No strong opinions, but why here and not on the repository?